src/index.js

```diff
@@ -71,7 +71,7 @@ class Plugin {
         };
 
         Object.freeze(filterBaseStatement); // Make it immutable
-
+        let principal = "logs." + this.serverless.service.provider.region + ".amazonaws.com";
```

Comment:
Can you please update this to use `const` and string templates?

```javascript
const principal = `logs.${this.serverless.service.provider.region}.amazonaws.com`;
```

If you can do that it will be GTM.